webapp/packages/plugin-navigation-tabs/src/NavigationTabs/NavigationTabsBar/shared/NavigationTabsBarTab.module.css

```diff
@@ -9,4 +9,5 @@
 .tab {
   composes: theme-ripple theme-background-background theme-text-text-primary-on-light from global;
   height: 38px !important;
+  border: 1px solid color-mix(in srgb, transparent, var(--theme-on-secondary) 10%);
```

Comment:
This way, we have doubled borders, I suggest setting only the right and bottom borders:
```
    border: 1px solid color-mix(in srgb, transparent, var(--theme-on-secondary) 10%);
    border-top: none;
    border-left: none;
```